charts/rasa-x/values.yaml

```diff
@@ -194,6 +199,53 @@ rasa:
       # - name: tmpdir
       #   emptyDir: {}
 
+# The database migration service requires Rasa X >= 0.33.0
+# db-migration-service specific settings
+dbMigrationService:
+  # settings related to the init-db container used as a init container for services
```

Comment:
Should we follow the recommended Helm best practices and repeat the property name at the start of each comment? https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml

Reply:
Sure good point 👍